snorkel/slicing/utils.py

```diff
@@ -16,6 +16,8 @@ def add_slice_labels(
     base_task: Task,
     slice_labels: np.ndarray,
     slice_names: List[str],
+    include_base: bool = True,
```

Comment:
Why do we need this?

Reply:
in our formulation, we treat the original task as a slice (trivial slice: all examples belong to the slice) so that we can combine representations from the original task as well. 

as a result, we need to add the base task's labels here as well

Reply:
right but it seems like we never have it be false?